src/main/scala/com/fulcrumgenomics/vcf/MakeTwoSampleMixtureVcf.scala

```diff
@@ -0,0 +1,200 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.vcf
+
+import java.util
+import java.util.Collections
+
+import com.fulcrumgenomics.FgBioDef._
+import com.fulcrumgenomics.cmdline.{ClpGroups, FgBioTool}
+import com.fulcrumgenomics.util.Io
+import com.fulcrumgenomics.vcf.MakeTwoSampleMixtureVcf._
+import dagr.sopt
+import dagr.sopt.{arg, clp}
+import htsjdk.samtools.util.IntervalList
+import htsjdk.variant.variantcontext._
+import htsjdk.variant.variantcontext.writer.{Options, VariantContextWriter, VariantContextWriterBuilder}
+import htsjdk.variant.vcf._
+
+import scala.collection.JavaConversions.asJavaCollection
+import scala.collection.JavaConversions.iterableAsScalaIterable
+import scala.collection.mutable.ListBuffer
+
+object MakeTwoSampleMixtureVcf {
+  val AlleleFractionField = "AF"
+  val GermlineFilter      = "alt_allele_in_normal"
+  val UnknownGtFilter     = "unknown_gt"
+  val MultiAllelicFilter  = "multi_allelic"
+
+  val TumorSampleName  = "tumor"
+  val NormalSampleName = "normal"
+}
+
+/**
+  * Creates a VCF by mixing two germline samples at a given proportion.
+  *
+  * @author Tim Fennell
+  */
+@sopt.clp(group=ClpGroups.VcfOrBcf, description=
+  """
+    |Creates a VCF by in-silico mixing genotypes from two samples. Writes a new VCF for a 'tumor' and
+    |optionally a 'normal' where the tumor is a mixture of two input samples, and the normal
+    |is one of the same input samples. Only outputs loci that are variant in the 'tumor'.
+    |Writes the 'AF' attribute in the format field with the expected allele fraction in the tumor.
```

Comment:
I've re-written the usage, but I thought it was clear from the name "MakeTwoSampleMixture" that the tumor genotypes were generated as a mixture of the two sets of incoming genotypes.